### DISCUSSION TITLE
Use multiple extensions in open dialog

### DISCUSSION THREAD

**Discussion by nonnorm** [tauri-apps/tauri#1134] [Q&A] [answered]
Hello,
Would it be possible to use multiple extensions in the filter section of the open dialog.  I've been trying, and I can't seem to get the right combination.

**Comment by nonnorm** [comment, marked as answer]
Okay, I figured it out.  You have to put semicolons between the values (e.g. `"pdf;jpg;jpeg"`)

**Reply by nothingismagick** [reply]
We should be better about showing how to do this in the docs.

**Reply by nklayman** [reply]
Updated with https://github.com/tauri-apps/tauri-docs/pull/153 :)
